pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -58,6 +58,9 @@ const (
 
 	// The expiration time of version cache.
 	versionCacheTTL = 60 * time.Second
+
+	// faild container max restart count
+	maxContainerRestartCount = 12
```

Comment:
why is this hardcoded to 12 and why do we need to limit this ?

Reply:
if not limit restart count, failed container will restart unlimited.  i think it is not necessary .

Reply:
？